lib/oxidized/input/ssh.rb

```diff
@@ -24,20 +24,23 @@ def connect node
       secure = Oxidized.config.input.ssh.secure
       @log = File.open(Oxidized::Config::Log + "/#{@node.ip}-ssh", 'w') if Oxidized.config.input.debug?
       port = vars(:ssh_port) || 22
+      
+      ssh_opts = {
+                :port => port.to_i,
+                :password => @node.auth[:password], :timeout => Oxidized.config.timeout,
+                :paranoid => secure,
+                :auth_methods => %w(none publickey password keyboard-interactive),
+                :number_of_password_prompts => 0,
+        }
+
       if proxy_host = vars(:ssh_proxy)
         proxy_command =  "ssh "
         proxy_command += "-o StrictHostKeyChecking=no " unless secure
         proxy_command += "#{proxy_host} -W %h:%p"
         proxy =  Net::SSH::Proxy::Command.new(proxy_command)
+        ssh_opts[:proxy] = proxy
```

Comment:
You can't pass an empty proxy in ssh_opts or you'll receive errors in the log. That's why I've changed it.